src/org/pentaho/reporting/platform/plugin/JobManager.java

```diff
@@ -80,6 +80,13 @@ public JobManager( final boolean isSupportAsync, final long pollingIntervalMilli
     return null;
   }
 
+  @GET @Path( "{job_id}/content" )
```

Comment:
Given that the method is the same (except for the logging) as the original getContent method, simply add the GET annotation to the other method. A REST endpoint can have multiple HTTP-method annotations.

This then eliminates this method and we dont have to write unit tests for another method.